frontend/app/_components/HomePageComponents/Footer.js

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+import {
+  FaTwitter,
+  FaFacebook,
+  FaRSS,
+  FaInstagram,
+  FaRss,
+} from "react-icons/fa";
+import styles from "./footer.module.css";
+import Image from "next/image";
+
+const Footer = () => {
+  return (
+    <footer className={styles.footer}>
+      {/* Top Section */}
+      <div className={styles.footerTop}>
+        <div className={styles.feature}>
+          <i className={styles.icon}>✔️</i>
+          <p>30 day money back guarantee</p>
+        </div>
+        <div className={styles.feature}>
+          <i className={styles.icon}>🌍</i>
+          <p>Support teams across the world</p>
+        </div>
+        <div className={styles.feature}>
+          <i className={styles.icon}>🔒</i>
+          <p>Safe & Secure online payment</p>
+        </div>
+      </div>
+
+      <hr className={styles.divider} />
+
+      {/* Middle Section */}
+      <div className={styles.footerMiddle}>
+        <div className={styles.column}>
+          <h3>WHO WE ARE</h3>
+          <ul>
+            <li>
+              <a href="#">About Us</a>
+            </li>
+            <li>
+              <a href="#">Team</a>
+            </li>
+            <li>
+              <a href="#">Work With Us</a>
+            </li>
+          </ul>
+        </div>
+        <div className={styles.column}>
+          <h3>COURSES</h3>
+          <ul>
+            <li>
+              <a href="#">Python</a>
+            </li>
+            <li>
+              <a href="#">PHP</a>
+            </li>
+            <li>
+              <a href="#">JavaScript</a>
+            </li>
+            <li>
+              <a href="#">React</a>
+            </li>
+            <li>
+              <a href="#">Nodejs</a>
+            </li>
+            <li>
+              <a href="#">eCommerce</a>
+            </li>
+            <li>
+              <a href="#">Machine Learning</a>
+            </li>
+            <li>
+              <a href="#">Angular</a>
```

Comment:
Should be moved to backend at some point, but okay for now. I would recommend you guys make a task for adding "categories" in the database. :D